editor/src/components/canvas/canvas-utils.spec.browser.ts

```diff
@@ -45,7 +45,7 @@ describe('updateFramesOfScenesAndComponents - pinFrameChange -', () => {
     )
 
     const pinChange = pinFrameChange(
-      TP.instancePath(TestScenePath, ['aaa', 'bbb']),
+      TP.appendNewElementPath(TestScenePath, ['aaa', 'bbb']),
```

Comment:
hah, this is a good trick that `TP.appendNewElementPath` has the same signature as instancePath :)